src/stateless/input.vue

```diff
@@ -568,6 +568,7 @@ export default {
         border: 0;
         outline: none;
         margin: 4px 0px;
+        transition: color @default-transition-time;
```

Comment:
Why was this change needed?

Reply:
We don't have it in master now, but it should be there

Reply:
So we broke master?

Reply:
Uh actually we don't need this now, this wouldn't change anything. Would have to add different colors on hover so it would make a difference.

Input is kinda strange now, for label text we have ```@gunpowder -> black``` on hover while the text entered in input is always black. This looks kinda strange next to selectboxes which have ```@gunpowder -> black``` both is selected or "Please select" state. We actually have it that way in designs, but should probably unify with other elements